src/SumoCoders/FrameworkCoreBundle/Composer/ScriptHandler.php

```diff
@@ -4,10 +4,73 @@
 
 use Composer\Script\Event;
 use Composer\IO\IOInterface;
+use SumoCoders\FrameworkCoreBundle\Installer\Installer;
 
 class ScriptHandler
 {
     /**
+     * Create the initial configuration for our project
+     *
+     * @param Event $event
+     */
+    public static function createInitialConfig(Event $event)
+    {
+        $installer = new Installer();
+        $io = $event->getIO();
+
+        // check if parameters.yml exists
+        $rootDir = realpath(__DIR__ . '/../../../../');
+        if (file_exists($rootDir . '/app/config/parameters.yml')) {
+            $io->write(
+                $installer->getDecoratedMessage(
+                    'Skipping creating the initial config as parameters.yml already exists',
+                    'info',
+                    $io->isDecorated()
+                )
+            );
+        } else {
+            $information = $installer->extractInformationFromPath($rootDir);
+
+            // ask all the information we need
+            $config = array();
+            $config['client'] = $installer->ask($io, 'client name', $information['client']);
+            $config['project'] = $installer->ask($io, 'project name', $information['project']);
+
+            $config['database_name'] = substr($config['client'], 0, 8) . '_' . substr($config['project'], 0, 7);
+            $config['database_user'] = $config['database_name'];
+
+            if ($information['is_local']) {
+                $config['database_host'] = '10.11.12.13';    // this is the ip-address of our Vagrantbox
```

Comment:
Gohja, wss wel. Maar dan verhuis je het probleem. Want op het punt dat dit script uitgevoerd wordt zijn er eigenlijk geen configs. Dat is net het punt van dit script om de initial config aan te maken.